### PR TITLE
Resprites the chocolate+berry cake

### PR DESCRIPTION
**Pull request by Greniza**
## About The Pull Request

Reflavors and resprites the chocolate and berry cake.

## Why It's Good For The Game

Adds more flavor to the chocolate and berry cake. New cake sprite:
piecake

The corresponding cake slice sprite was changed accordingly.

## Changelog
:cl:
imageadd: new sprites and flavor for the chocolate and berry cake
/:cl:

### CODE REVIEW

**Comment by maxymax13** [comment]
this pr is a lie

**Comment by Twaticus** [comment]
this only changes the code for the cake slice and not the cake itself, unless im misreading it. also pls change the icon name to match

**Review by Twaticus — APPROVED** [review]

**Comment by Mothblocks** [comment]
its so hard to find the geeks and gamers in the coderbus

File: code/game/objects/items/food/cake.dm [hunk 1]
```diff
@@ -317,10 +317,10 @@
 	tastes = list("blackberry" = 2, "strawberries" = 2, "vanilla" = 2, "sweetness" = 2,"cake" = 3)
 	foodtypes = GRAIN | DAIRY | FRUIT | SUGAR
 
-/obj/item/food/cake/bscc // blackbarry strawberries chocolate cake
-	name = "blackberry and strawberry chocolate cake"
-	desc = "A chocolate cake, filled with assortment of blackberries and strawberries!"
-	icon_state = "blackbarry_strawberries_cake_coco_cake"
+/obj/item/food/cake/bscc // blackbarry strawberries chocolate cake <- this is a relic from before resprite
```
[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/game/objects/items/food/cake.dm:320]: i was tired

**Pull request merged**